packages/yew/src/html/component/lifecycle.rs

```diff
@@ -255,6 +255,7 @@ mod tests {
     #[derive(Clone, Properties, Default)]
     struct Props {
         lifecycle: Rc<RefCell<Vec<String>>>,
+        #[cfg(feature = "wasm_test")]
         create_message: Option<bool>,
```

Comment:
```
    Checking yew-stdweb v0.17.4 (/home/runner/work/yew/yew/packages/yew-stdweb)
error: field is never read: `create_message`
Error:    --> src/html/component/lifecycle.rs:258:9
    |
258 |         create_message: Option<bool>,
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: `-D dead-code` implied by `-D warnings`
```

I had to do this because this field has become optional with the change of this PR on fields that are Option.

I'm not if this whole takes makes sense without a browser? If it doesn't, I can change the cfg so the test will be run only for `wasm_test`.

@jstarry can you help me on this one? Should I keep it with the cfg flag like I did or should I disable the entire test to run outside browser?